sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java

```diff
@@ -204,6 +205,18 @@ public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
     }
   }
 
+  protected static final class DecimalCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return Values.decimal(value);
+    }
```

Comment:
Do we really need this method?
It looks like every implementation is going to be a direct call to the corresponding `Values.<typeName>` method.

Reply:
I did not check that, but if it really is `value.getDecimal()`, that'd be better?

Reply:
I collapsed the method